metadata_service/api/user.py

```diff
@@ -192,12 +204,12 @@ def get(self, user_id: str) -> Iterable[Union[Mapping, int, None]]:
         :return:
         """
         try:
-            resources = self.client.get_table_by_user_relation(user_email=user_id,
-                                                               relation_type=UserResourceRel.read)
+            resources = self.client.get_frequently_used_tables(user_email=user_id)
```

Comment:
hey @jinhyukchang , you need to add this call to base_proxy and atlas_proxy as well.

Reply:
Got it and added.